lab-anthony/.eslintrc

```diff
@@ -0,0 +1,43 @@
+{
```

Comment:
Try to be consistent with your formatting. You're using trailing commas all over in your webpack config, but nowhere else in your code. The point of an eslintrc is to enforce consistentcy throughout a project. Also, if you are going to use trailing commas, I would have argued that the frontend code would would be where to use them (where babel will handle them appropriately), rather than in the webpack config, which will not be sent through babel.